tools/fw-logger/rs-fw-logger.cpp

```diff
@@ -70,13 +72,64 @@ int main(int argc, char* argv[])
     context ctx;
     device_hub hub(ctx);
 
+    rs2::device_list all_device_list = ctx.query_devices();
+    if (all_device_list.size() == 0) {
+        std::cout << "\nLibrealsense is not detecting any devices" << std::endl;
```

Comment:
The application should wait for devices to connect - this new constrain breaks the UC.